src/runtime/opencl/opencl_device_api.cc

```diff
@@ -24,6 +24,8 @@ void OpenCLWorkspace::SetDevice(TVMContext ctx) {
 
 void OpenCLWorkspace::GetAttr(
     TVMContext ctx, DeviceAttrKind kind, TVMRetValue* rv) {
+  //Initialize RV to a known value!
+  *rv = 0;
```

Comment:
this is not a desirable approach, since most cases we do prefer not to set it(which returns None)